main/runserver.sh

```diff
@@ -59,6 +59,7 @@ echo "export API_FQDN=\"$API_FQDN\"" >> $HOME/.env
 echo "export FRONTEND_URL=\"$FRONTEND_URL\"" >> $HOME/.env
 echo "export PRODUCTION=\"$PRODUCTION\"" >> $HOME/.env
 echo "export TEST_EMAILS=\"$TEST_EMAILS\"" >> $HOME/.env
+echo "export DEBUG_INTERNAL_IP=\"/sbin/ip route|awk '/default/ { print $3 }'\"" >> $HOME/.env
```

Comment:
My recommendation was to add a new script called `run_develop.sh` which be used to run a development version only. And add this logic there.
Something like this

**main/run_develop.sh** [add this to **docker-compose.yml** by replacing `python3 manage.py runserver`]
```sh
#!/bin/bash
export DEBUG_INTERNAL_IP=`/sbin/ip route| awk '/default/ { print $3 }'`
python3 manage.py runserver
```